app-modules/notification/src/DataTransferObjects/EmailChannelResultData.php

```diff
@@ -42,7 +42,10 @@ class EmailChannelResultData extends NotificationResultData
 {
     public function __construct(
         public bool $success,
+        public array $recipients = [],
         public ?SentMessage $message = null,
         public ?string $error = null,
-    ) {}
+    ) {
+        parent::__construct();
```

Comment:
Just for my edification, why is this necessary?

Reply:
It's just best practice for an extension to call `parent::__construct();` to make sure the object is being instantiated correctly. It's not always needed and there are times when maybe you wouldn't want it. But PHPStorm and some static analysis will yell at you without.